go.mod

```diff
@@ -7,20 +7,20 @@ require (
 	github.com/aws/aws-sdk-go-v2 v1.36.3
 	github.com/fsnotify/fsnotify v1.8.0
 	github.com/gofrs/flock v0.12.1
-	github.com/google/go-cmp v0.6.0
+	github.com/google/go-cmp v0.7.0
 	github.com/manifoldco/promptui v0.9.0
-	github.com/prometheus/client_golang v1.20.5
+	github.com/prometheus/client_golang v1.22.0
 	github.com/sirupsen/logrus v1.9.3
 	github.com/spf13/afero v1.12.0
 	github.com/spf13/cobra v1.8.1
 	github.com/spf13/viper v1.19.0
 	golang.org/x/time v0.9.0
 	gopkg.in/yaml.v2 v2.4.0
-	k8s.io/api v0.32.0
-	k8s.io/apimachinery v0.32.0
-	k8s.io/client-go v0.32.0
-	k8s.io/code-generator v0.32.0
-	k8s.io/component-base v0.32.0
+	k8s.io/api v0.33.0
+	k8s.io/apimachinery v0.33.0
+	k8s.io/client-go v0.33.0
+	k8s.io/code-generator v0.33.0
+	k8s.io/component-base v0.33.0
 	k8s.io/sample-controller v0.26.1
```

Comment:
if we update this dependency, then dependabot will be able to automatically handle most of the dependency updates. updating `k8s.io/sample-controller` is not trivial, but it needs to be taken care of given how far behind its falling

Reply:
https://pkg.go.dev/k8s.io/sample-controller/pkg/apis/samplecontroller?tab=versions
Updated to match with latest.

Reply:
thanks for updating that @Ganiredi !